v8go.cc

```diff
@@ -200,26 +200,296 @@ void ContextDispose(ContextPtr ptr) {
 
 /********** Value **********/
 
+#define LOCAL_VALUE(ptr) \
+    m_value* val = static_cast<m_value*>(ptr); \
+    m_ctx* ctx = val->ctx_ptr; \
+    Isolate* iso = ctx->iso; \
+    Locker locker(iso); \
+    Isolate::Scope isolate_scope(iso); \
+    HandleScope handle_scope(iso); \
+    Context::Scope context_scope(ctx->ptr.Get(iso)); \
+    Local<Value> value = val->ptr.Get(iso);
+
 void ValueDispose(ValuePtr ptr) {
-  delete static_cast<m_value*>(ptr);
+    delete static_cast<m_value*>(ptr);
```

Comment:
would love to get clangtidy/otherwise in here to standardize the non-go code formatting as well.

Reply:
Agree; will save that for a separate PR; I did format the whole file, but I thought that was too much of a change to add to this PR, so I reverted that. (missed this line though).

Reply:
#51